### PR TITLE
Add doc note for disabling CSRF secure cookie from the command line

### PR DESCRIPTION
**Pull request by eddycharly**
# Changes

This PR adds a note in the development docs to show how to disable CSRF secure cookie.

/cc @CarolynMabbott 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by a-roberts** [comment]
@CarolynMabbott do you feel there's enough here then? Let us know and then let's get this in

**Comment by eddycharly** [comment]
FWIW i created a separate issue to add supported arguments docs https://github.com/tektoncd/dashboard/issues/1404

I will try to address it this afternoon

**Comment by eddycharly** [comment]
Complementary PR https://github.com/tektoncd/dashboard/pull/1408

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ kustomize build overlays/dev-openshift-imagestream --load_restrictor=LoadRestric
 The dashboard can be accessed by running `kubectl proxy`, which proxies requests through the API Server. You can access the web UI at `localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/`
 
 Important note: the setting for a CSRF secure cookie must be *false* if you intend to access the Dashboard pod through port-forward. Otherwise, it should be true (for example, when using a Route, or Ingress, which is secured with TLS).
+You can disable CSRF secure cookie by adding `--csrf-secure-cookie=false` in the command line args when starting the dashboard backend.
```
Comment (AlanGreene) [inline DEVELOPMENT.md:148]: 'You can disable the Secure attribute on the CSRF cookie...'

To make sure it's clear we're not disabling the cookie, just changing one of its attributes.

Comment (eddycharly) [reply DEVELOPMENT.md]: fixed

**Pull request merged**